### ISSUE TITLE
[ResNet50/Pytorch] loss is inf when training resnet50 with 2 Nodes, each have 8 GPUs

### ISSUE THREAD

**Issue by elevenxiang** [NVIDIA/DeepLearningExamples#1266]
Related to **ResNet50/Pytorch** 

**Describe the bug**
got loss is inf when training resnet50 on 2 Nodes, each have 8 GPUs

RUNNING EPOCHS FROM 0 TO 90
Process 0 Worker 0 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 1 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 2 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 3 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 4 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 5 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 6 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 7 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 8 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
Process 0 Worker 9 set affinity to: {0, 1, 2, 3, 4, 5, 6, 7, 64, 65, 66, 67, 68, 69, 70, 71}
DLL 2023-02-15 08:27:46.754175 - Epoch: 0 Iteration: 20  train.loss : **inf** None train.compute_ips : 32893.73 images/s train.total_ips : 32709.71 images/s train.lr : 0.256  train.grad_scale : 128.00000 None
DLL 2023-02-15 08:28:13.202587 - Epoch: 0 Iteration: 40  train.loss : **inf** None train.compute_ips : 30845.63 images/s train.total_ips : 30460.06 images/s train.lr : 0.256  train.grad_scale : 128.00000 None

**To Reproduce**
Node 0:
python ./multiproc.py \
--master_addr xx.xx.xx.xx --master_port 29501 \
--nnodes 2 --nproc_per_node 8 --node_rank 0 \
./launch.py --model resnet50 --precision AMP --mode convergence --no-checkpoints \
--platform DGXA100 /imagenet \
--epochs 90 --mixup 0.0 --batch-size 256 \
--workspace ${1:-./} --raport-file raport.json

Node 1:
python ./multiproc.py \
--master_addr xx.xx.xx.xx --master_port 29501 \
--nnodes 2 --nproc_per_node 8 --node_rank 1 \
./launch.py --model resnet50 --precision AMP --mode convergence --no-checkpoints \
--platform DGXA100 /imagenet \
--epochs 90 --mixup 0.0 --batch-size 256 \
--workspace ${1:-./} --raport-file raport.json

**Expected behavior**
expect have normal loss value, and it will decrease for each iteration

**Environment**
Please provide at least:
* Container version (e.g. pytorch:19.05-py3): nvcr.io/nvidia/pytorch:22.04-py3 
* GPUs in the system: (e.g. 8x Tesla V100-SXM2-16GB): A100 80G
* CUDA driver version (e.g. 418.67): Driver Version: 470.57.02    CUDA Version: 11.6

**Comment by elevenxiang** [comment]
it will be ok, if I change to 2 Nodes, each Node use 4 GPUs

**Comment by fallfo** [comment]
try --optimizer--batch-size 4096

**Comment by elevenxiang** [comment]
> try --optimizer--batch-size 4096

thank you so much, it works
